strings/reverse.go

```diff
@@ -0,0 +1,10 @@
+package strings
+
+func ReverseString(input string) string {
```

Comment:
I forgot to tell you in the previous review, but can you rename this to just `Reverse`. Since our package is called `strings` on usage it looks like `strings.ReverseString(..)` which looks bad. `strings.Reverse(..)` is cleaner and more go-like